tests/test_client.py

```diff
@@ -103,6 +105,7 @@ def setUpClass(cls):
         else:
             cls.client = Client(
                 CONFIG["openid_configuration"], CONFIG['client_id'],
+                http_client=MinimalHttpClient(),
```

Comment:
Either remove the `http_client=`, or you might as well try to copy [this one](https://raw.githubusercontent.com/AzureAD/microsoft-authentication-library-for-python/oauth2-session/tests/test_client.py). Choose an approach you like better.